src/annotations/components/annotationForm/AnnotationStartTimeInput.tsx

```diff
@@ -30,6 +46,24 @@ export const AnnotationStartTimeInput: FC<Props> = (props: Props) => {
     }
   }
 
+  const isInputValueInvalid = (): boolean => {
+    if (inputValue === null) {
+      return false
+    }
+
+    return !isValidTimeFormat(inputValue)
+  }
+
+  const inputErrorMessage = (): string | undefined => {
```

Comment:
let's rename this to `getInputValidationMessage` or something like that. Error message implies there _is_ an error when there may not be. It also shouldn't return undefined, it'll only return a string according to this method implementation.

the same comment about passing in an argument applies here:

```ts
const getInputValidationMessage = (inputValue: string): string => {
  if (!isInputValueValid(inputValue)) {
    return ...
  }
  return ''
}

```ts

Reply:
Done ✅